src/components/Onboarding/OnboardingScreen.js

```diff
@@ -0,0 +1,668 @@
+    setCurrentStep(currentStep + 1);
+  }, [currentStep, role]);
+
+  const isInvalid = useMemo(() => (
+    formState.password !== formState.confirmPassword
+      || formState.password === ''
+      || formState.email === ''
+  ),
+  [formState]);
+
+  const repeatPwdCheck = useMemo(() => {
+    let msg = '';
+    if (formState.password === '' && formState.confirmPassword === '') {
+      msg = '';
+    } else if (formState.password !== '' && formState.confirmPassword !== '') {
+      if (formState.password !== formState.confirmPassword) {
+        msg = "Passwords don't match!";
+      }
+    }
+    return msg;
+  },
+  [formState]);
+
+  const step3Check = useMemo(() => {
+    let msg = '';
+    if (formState.zipcode !== '' && Number.isNaN(formState.zipcode)) {
+      msg = 'Location (zipcode) should hold a number!';
+    }
+    return msg;
+  },
+  [formState]);
+
+  const step3IsInvalid = useMemo(() => (
+    formState.zipcode === '' || formState.firstName === ''
+      || formState.lastName === '' || formState.org === '' || formState.phone === ''
+      || formState.zipcode === ''
+  ),
+  [formState]);
+
+  function Step1() {
+    if (currentStep !== 1) {
+      return null;
+    }
+    return [
+      <div className={classes.subtitleText}>
+        Are you a...
+      </div>,
+      <Grid
+        container
+        spacing={2}
+        direction="row"
+        justify="center"
+        alignItems="stretch"
+      >
+        {BUTTONS.map((bt) => (
+          <Grid
+            item
+            className={classes.card}
+            component={Card}
+            xs={12}
+            sm={6}
+            md={3}
+            key={BUTTONS.indexOf(bt)}
+          >
+            <CardActionArea onClick={roleChange} value={bt.id}>
+              <CardMedia
+                image={bt.image}
+                style={bt.fruit}
+              />
+              <CardContent>
+                <Typography className={classes.labelText}>
+                  {bt.title}
+                </Typography>
+              </CardContent>
+            </CardActionArea>
+          </Grid>
+        ))}
+      </Grid>,
+    ];
+  }
+
+  function Step4() {
+    if (currentStep !== 4) {
+      return null;
+    }
+    return [
+      <div>
+        <div className={classes.cenSubtitleText}>
+          Account Created!
+        </div>
+        <div className={classes.cenRegSubtitleText}>
+          { role === 1 ? 'Please await authentication from Farm2People.'
+            : 'Farm2People will be in touch with you shortly.'}
+        </div>
+      </div>,
+      <Grid
+        container
+        spacing={2}
+        alignItems="center"
+      >
+        <Grid item xs={12}>
+          <Box mt={5}>
+            <Button
+              type="button"
+              className={classes.submitButton}
+              color="primary"
+              variant="contained"
+              style={{ backgroundColor: '#53AA48' }}
+              onClick={onPrev}
```

Comment:
Why does the submit button take us to the previous step? Let's implement it so that way it redirects us to the landing screen using history.push() --> I have this code in my Navbar and SignInScreen